telegram/src/main/kotlin/com/github/kotlintelegrambot/updater/Looper.kt

```diff
@@ -12,24 +17,23 @@ internal interface Looper {
  * intended to run the loop in a different thread). The loop will stop if the thread running the
  * loop is interrupted or in the next iteration after the [quit] method is called.
  */
-internal class ExecutorLooper(
-    private val loopExecutor: Executor,
-) : Looper {
+internal class ExecutorLooper(ioDispatcher: CoroutineDispatcher) : Looper {
```

Comment:
Not important. The name of this class and the comment seems a bit outdated now that we are using a `CoroutineDispatcher` rather than an `Executor`.